### PR TITLE
Preparing the repository for seperated dart and flutter package

### PR DESCRIPTION
**Pull request by fischerscode**
Fixes: #78 #408

TODOs:
- [x] create package directory and move code
- [ ] Docs
- [ ] dart example
- [ ] migration instructions

See the conversation in #434 for more information.

### CODE REVIEW

**Comment by maaeps** [comment]
The checks obviously fail, as all files have been moved.

**Comment by fischerscode** [comment]
@phillwiggins Just to avoid confusion: I have renamed my github account from @maaeps to @fischerscode

**Comment by nstrelow** [comment]
Awesome work!!!

File: packages/dart/pubspec.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  xxtea: ^2.0.3
+
+  # Utils
+  uuid: ^2.2.2
+  meta: ^1.1.8
+  path: ^1.7.0
+
+dev_dependencies:
+  # Testing
+  flutter_test:
```
Comment (nstrelow) [inline packages/dart/pubspec.yaml:27]: You can go back to using test here

**Pull request merged**